doc/index.rst

```diff
@@ -184,10 +193,15 @@ The included ``.nc`` files are PyPSA network files which can be imported with Py
 Licence
 =======
 
-The code in PyPSA-Eur is released as free software under the `GPLv3
-<http://www.gnu.org/licenses/gpl-3.0.en.html>`_, see
-`LICENSE <https://github.com/PyPSA/pypsa-eur/blob/master/LICENSE.txt>`_.
-However, different licenses and terms of use apply to the various input data, which are summarised below.
+PyPSA-Eur work is released under multiple licenses:
+
+* All original source code is licensed as free software under `GPL-3.0-or-later <LICENSES/GPL-3.0-or-later.txt>`_.
+* The documentation is licensed under `CC-BY-4.0 <LICENSES/CC-BY-4.0.txt>`_.
+* Configuration and data files are mostly licensed under `CC0-1.0 <LICENSES/CC0-1.0.txt>`_.
```

Comment:
According to the [dep5 file](https://github.com/PyPSA/pypsa-eur/blob/license/.reuse/dep5) all files in `data` are licenses as CC-BY-4.0.
How about separating it?

```suggestion
* Configuration files are mostly licensed under `CC0-1.0 <LICENSES/CC0-1.0.txt>`_.
* Data files are licensed under `CC-BY-4.0 <LICENSES/CC-BY-4.0.txt>`_.
```